src/board.cpp

```diff
@@ -40,14 +40,22 @@ void setupHW()
   Display::initM5();
   M5.update();
 #else
+  #ifdef ePaperPowerPin
```

Comment:
Could be a machine without this pin? I would rather use the logic like: remove else and use ifndef